tests/test_app.py

```
@@ -0,0 +1,102 @@
+        mock_path_instance.__truediv__ = MagicMock(return_value=mock_log_dir)
+
+        with patch("app.DEBUG", False):
+            configure_logging()
+            # Check that setup_logging was called with INFO level
+            call_args = mock_setup_logging.call_args
+            assert call_args[1]["level"] == logging.INFO
+
+    @patch("app.setup_logging")
+    @patch("app.Path")
+    @patch.dict("os.environ", {}, clear=False)
+    def test_configure_logging_uses_default_main_dir(self, mock_path_class, mock_setup_logging):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_setup_logging`

(ARG002)